docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+  {% endif %}
+    <div class="container position-relative px-4 px-lg-5">
+      <div class="row gx-4 gx-lg-5 justify-content-center">
+        <div class="col-md-10 col-lg-8 col-xl-7">
+          <div class="site-heading">
+            <h1>{{ data.heading }}</h1>
+            <span class="subheading">{{ data.subheading }}</span>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+{% endfragment %}
+```
+
+<template v-slot:caption>
+views/fragments/header.html
+</template>
+</AposCodeBlock>
+
+We will revisit the field schema for the header when we create our default page.
+
+## Modifying the `layout.html` file
+
+Now that we have our three common fragments setup, we need to start adding them to our pages. Again, the navigation and footer fragments are populated with data from the global settings. Therefore, we can add those fragments to our base layout file - `views/layout.html`. In addition, looking at the head section of the template files, we can see that the `fontawesome` and Google `Lora` and `Open Sans` fonts are being added. We can alter this same file to load those files into the head section
+
+::: note
+If your pages are being loaded and viewed in an EU country, serving fonts from Google might violate GDPR. There are multiple articles on the internet detailing how to download the files and host them locally in order to comply with GDPR.
+:::
+
+The first modification we are going to make is to import our two fragments.
+
+<AposCodeBlock>
+```django
+{% import 'fragments/navigation.html' as navigation %}
+{% import 'fragments/footer.html' as footer %}
+```
+
+</AposCodeBlock>
+
+This will bring our fragments into the template and name them `navigation` and `footer`.
+
+To load our font files we are going to take advantage of the `extraHead` section of the `outerLayout` template that our `layout.html` is extending. You can read the [documentation](https://v3.docs.apostrophecms.org/guide/layout-template.html#layout-templates) to learn about other sections of this template that can be extended. Add the `{% block extraHead %}{% endblock %}` tags after the `{% block title%}` section. Within those tags copy the fonts section of the head from any of the template pages. You can also see that there are links for the site favicon and some other meta tags in this section of the templates. You can elect to add those if you desire.
+
+Finally, delete all of the markup in the `beforeMain` and `afterMain` sections. into the `beforeMain` block render the navigation - `{% render navigation.navigationArea() %}`. In the `afterMain` block render the footer fragment = `{% render footer.footerArea() %}`. The final modification we can make to the template is to add the semantic `<main></main>` tags. Looking at the original template pages, we can see that the main section has a class of `mb-4`. This markup can be added around the `{% block main %}{% endblock %}` tags.
```

Comment:
```suggestion
Finally, delete all of the markup in the `beforeMain` and `afterMain` sections. In the `beforeMain` block, render the navigation - `{% render navigation.navigationArea() %}`. In the `afterMain` block render, the footer fragment - `{% render footer.footerArea() %}`. The final modification we can make to the template is to add the semantic `<main></main>` tags. Looking at the original template pages, we can see that the main section has a class of `mb-4`. This markup can be added around the `{% block main %}{% endblock %}` tags.
```
Formatting.

> This markup can be added around the `{% block main %}{% endblock %}` tags.

Huh. I usually place it within the `main` block. Didn't think to put it outside. Nice.